rt_utils/rtstruct.py

```diff
@@ -25,6 +25,18 @@ def set_series_description(self, description: str):
         """
 
         self.ds.SeriesDescription = description
+        
+    def roi_number(self):        
+         ROI_list = self.ds.StructureSetROISequence
+         ROI_number = []
+         for i in range(len(ROI_list)):
+                ROI_number.append(ROI_list[i].ROINumber.numerator)
+         j = 0 
+         while j in ROI_number :
+                j = j+1
+         return j             
+    
+    """choose a ROI number which has not been attributed to the ROI already existing the original RTstructure. This might prevent duplicate and error in opening"""
```

Comment:
Comment should be placed after the function declaration

```
def roi_number(self):
    """
    choose a ROI number which has not been attributed to the ROI already existing the original RTstructure. This might prevent duplicate and error in opening
    """